gestalt/__init__.py

```diff
@@ -159,6 +159,16 @@ def build_config(self) -> None:
         self.__parse_dictionary_keys(self.__conf_sets)
         self.__conf_sets = self.__interpolate_keys(self.__conf_sets)
 
+    def combine_into(
```

Comment:
should we put this into a separate `utils.py` package? I'm not sure this really belongs in the `Gestalt` class

Reply:
@john-delivuk what do you think?

Reply:
I made it a public function which is now accessible outside of the gestalt class. I think it's better there than a util function. We can choose to make a restructure at a later date as a minor update if we decide on adding more features that belong to a util package.